lib/realtime/server.ex

```diff
@@ -179,6 +183,19 @@ defmodule Realtime.Server do
   def handle_info({reference, _}, %__MODULE__{} = state) when is_reference(reference),
     do: {:noreply, state}
 
+  def handle_info(:check_data_status, %__MODULE{ets: ets} = state) do
+    all_vehicles =
+      {ets, :all_vehicles}
+      |> lookup()
+      |> Enum.filter(fn vehicle_or_ghost -> match?(%Vehicle{}, vehicle_or_ghost) end)
+
+    data_status_fn = Application.get_env(:skate, :data_status_fn) || (&DataStatus.data_status/1)
```

Comment:
I believe that `get_env` takes a default argument third, so `data_status_fn = Application.get_env(:skate, :data_status_fn, &DataStatus.data_status/1)`